README.md

```diff
@@ -90,6 +90,15 @@ any change.
 The increment of the patch version indicates there is no change of the supported
 languages and no change of the existing APIs.
 
+## Development
+
+To set up a development environment and run specs do
+
+```bash
+git submodule init && git submodule update
```

Comment:
I don't think you should note about submodule here because you should do `git clone --recursive https://github.com/akihikodaki/cld3-ruby.git` when you clone. (By the way, if you forgot to clone or you want to check `.gitmodules` beforehand, you can `git submodule update --init`.)

Reply:
👍 fair point will remove the second commit